index.html

```diff
@@ -525,6 +525,96 @@ <h3 class="hosting__title">Wybierz swój plan hostingowy</h3>
 					/>
 				</ul>
 			</section>
+			<section class="resources">
+				<ul>
```

Comment:
Masz zle uzyta liste. Jedynym elementem zagniezdzonym w `ul` moze byc `li`, czyli cos takiego

```
<ul>
  <li>
     <ul>
        <li></li>
    </ul>
  </li
</ul>
```

Reply:
Tak samo, to

```
<ul class="resources__text-bold">
  <strong>Pełne Bezpieczeństwo</strong>
</ul>
```

Takie cos nie jest prawidlowym htmleme.

Reply:
Zobacz sobie jak oni to zrobli